outputs.tf

```diff
@@ -3,3 +3,8 @@ output "drata_service_account_key" {
   description = "Service Account Key"
   sensitive   = true
 }
+
+output "drata_service_account" {
+  value       = google_service_account.drata
```

Comment:
let's adjust this output in order to show the email only
```suggestion
  value       = google_service_account.drata.email
```

Reply:
@tonygc is this looking ok now?